components/CardNews/src/index.scss

```diff
@@ -0,0 +1,90 @@
+.denhaag-card-news {
+  text-decoration: var(--denhaag-card-news-text-decoration);
+}
+
+.denhaag-card-news__figure {
+  border: var(--denhaag-card-news-figure-border);
+  padding-block-start: var(--denhaag-card-news-figure-padding);
+  padding-block-end: var(--denhaag-card-news-figure-padding);
+  padding-inline-start: var(--denhaag-card-news-figure-padding);
+  padding-inline-end: var(--denhaag-card-news-figure-padding);
+  margin-inline-start: var(--denhaag-card-news-figure-margin);
+  margin-inline-end: var(--denhaag-card-news-figure-margin);
+  margin-block-start: var(--denhaag-card-news-figure-margin);
+  margin-block-end: var(--denhaag-card-news-figure-margin);
+  min-width: var(--denhaag-card-news-figure-min-width);
+  width: var(--denhaag-card-news-figure-width);
+  max-width: var(--denhaag-card-news-figure-max-width);
+  min-height: var(--denhaag-card-news-figure-min-height);
+  height: var(--denhaag-card-news-figure-height);
+  max-height: var(--denhaag-card-news-figure-max-height);
+}
+
+.denhaag-card-news--hover .denhaag-card-news__figure {
+  box-shadow: var(--denhaag-card-news-figure-box-shadow);
+}
+
+.denhaag-card-news--focus .denhaag-card-news__figure {
+  box-shadow: none;
+  outline: var(--denhaag-card-news-figure-outline);
+}
+
+.denhaag-card-news:hover .denhaag-card-news__figure {
+  box-shadow: var(--denhaag-card-news-figure-box-shadow);
+}
+
+.denhaag-card-news:focus .denhaag-card-news__figure {
```

Comment:
Why the duplication when u can just do a comma ;-) 

`.denhaag-card-news::focus .denhaag-card-news__figure,
.denhaag-card-news--focus .denhaag-card-news__figure {}`

Reply:
@paulAcato because the storybook linter

Reply:
@robbieatacato , weird.. I do this all the time..

@bddjong, did there something change that Robbie can't seperate the classNames by comma? And therefor must write duplicate CSS (more chance for issues in the future).

Reply:
The rule Robbie is referring to is the stylelint `no-descending-specificity` rule. In a few other components, this rule was ignored by using this code:

```css
/* stylelint-disable-next-line no-descending-specificity */
.denhaag-card--focus .denhaag-card__arrow,
.denhaag-card:focus-within .denhaag-card__arrow {
  outline: var(--denhaag-focus-border);
}
```

It's caused by `:` selectors having a higher specificity than normal bem classes.